project-base/tests/ShopBundle/Database/Model/Product/ProductRepositoryTest.php

```diff
@@ -166,14 +166,14 @@ public function testOrderingByProductPriorityInSearch()
         $product1 = $this->getReference(ProductDataFixture::PRODUCT_PREFIX . 1);
         $product2 = $this->getReference(ProductDataFixture::PRODUCT_PREFIX . 45);
 
-        $this->setProductOrderingPriority($product1, 0);
-        $this->setProductOrderingPriority($product2, 1);
+        $this->setProductOrderingPriority($product1, 2);
+        $this->setProductOrderingPriority($product2, 3);
 
         $results = $this->getProductsForSearchOrderedByPriority('sencor');
         $this->assertSame($product2, $results[0]);
         $this->assertSame($product1, $results[1]);
 
-        $this->setProductOrderingPriority($product2, -1);
+        $this->setProductOrderingPriority($product2, 1);
 
```

Comment:
Why is this test updated? I thought that the behaviour of product ordering will be the same.

Reply:
I have to explain it in the commit message.

The problem is that searching now return more results, so we had to increate the priority to have these products always on top of others.